packages/smooth_app/lib/pages/onboarding/knowledge_panel_page_template.dart

```diff
@@ -109,4 +115,47 @@ class _KnowledgePanelPageTemplateState
           );
         });
   }
+
+  List<Widget> _buildHintPopup() {
+    Widget hintPopup = InkWell(
```

Comment:
final tag to be used online
```suggestion
  final  Widget hintPopup = InkWell(
```

Reply:
✅